src/components/article/ImageDiff.js

```diff
@@ -38,22 +49,40 @@ class ImageDiff extends FitwidthMixin(Component) {
 
   render() {
     let imageByDevice = _.get(this.props, [ 'content', 0 ], {})
-    let { mobile, tablet, desktop } = imageByDevice
-    const imgStyle = {
-      width: this.state.width
+    let { mobile, tablet, desktop, original } = imageByDevice
+    let { isMounted, screenType, width, percentage, onhovered } = this.state
+    const height = this._getHeight(width, original, width, width)
+    let buttonClass = null
+    let outerStyle = {
+      width: width,
+      minHeight: height
+    }
+    let imgStyle = {
+      ...outerStyle,
+      height: height
+    }
+
+    if(onhovered) {
+      buttonClass = styles.hovered
     }
 
     return (
-      <div ref="imgDiff">
+      <div ref="imgDiff" className={styles.diffContainer} style={outerStyle}>
         <figure className={styles.wrapper}>
           <div className={styles.imgContainer}>
-            <img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/photoshop-face-before.jpg" style={imgStyle}/>
+            <img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/photoshop-face-after.jpg" style={imgStyle}/>
           </div>
-          <div>
-            <img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/photoshop-face-before.jpg"/>
+          <div className={styles.overlayContainer} style={{ width: percentage+'%' }}>
```

Comment:
yes. I will replace the url as the new version of imagediff of Keystone is being deployed.